README.md

```diff
@@ -4,11 +4,33 @@ Gets downloaded for the `os-upgrade.service` in the cluster.
 
 To run the upgrade of the cluster:
 
-1. Run `os-upgrade.service`:
+0. If you are running this service in prod you should first fail over read traffic to the other cluster by ssh-ing to that cluster and running
```

Comment:
As per Sarah's mail we have to submit a CR prior of doing any config change, but I don't know if that have to be documented...